tests/e2e/pageObjects/browser-page.ts

```diff
@@ -127,6 +128,9 @@ export class BrowserPage {
     formatSwitcher = Selector('[data-testid=select-format-key-value]', { timeout: 2000 });
     formatSwitcherIcon = Selector('img[data-testid^=key-value-formatter-option-selected]');
     selectIndexDdn = Selector('[data-testid=select-index-placeholder],[data-testid=select-search-mode]', { timeout: 1000 });
+    createIndex = Selector('[data-testid=create-index-btn]');
```

Comment:
I suggest to rename it to 'createIndexBtn'

Reply:
and the same is for buttons below

Reply:
fixed